pycloudlib/instance.py

```diff
@@ -34,8 +34,10 @@ def __init__(self, key_pair):
         self.key_pair = key_pair
         self.port = '22'
         self.username = 'ubuntu'
+        self.connect_timeout = 60
 
     @abstractproperty
+    @property
```

Comment:
Thanks for adding the `@property` decorator! Per https://docs.python.org/3/library/abc.html#abc.abstractproperty, this should be:

```suggestion
    @property
    @abstractmethod
```

because `abstractproperty` has been deprecated since 3.3 (which predates even trusty, which is 3.4).